src/app/shared/models/dot-apps/dot-apps.model.ts

```diff
@@ -35,3 +36,8 @@ export interface DotAppsSaveData {
         value: string;
     };
 }
+
+interface LabelValueOption {
```

Comment:
We use the interface `SelectItem` from primeNg in most components. Should we changed ?